docs/pipeline_creation.md

```diff
@@ -0,0 +1,336 @@
+# Creating a pipeline
+
+This creating a pipeline document outlines the steps required to create a pipeline compatible with the data platform. This document will go into detail of steps 1-7 (all steps) in the general pipeline document. It will outline how to achieve a data spanning several years across multiple LAs as highlighted in step 7, but will not include specific/bespoke analyses.
+
+## 1. Create a new pipeline folder with the following subfolders (example files have also been included)
+
+```
+liia_tools/
+├─ new_pipeline/
+│  ├─ __init__.py
+│  ├─ stream_filters.py (only for xml)
+│  ├─ stream_pipeline.py
+│  ├─ stream_record.py (only for xml)
+│  ├─ spec/
+│  │  ├─ __init__.py
+│  │  ├─ pipeline.json
+│  │  ├─ new_data_schema_2024.yml or new_data_schema_2024.xsd
```

Comment:
out of curiosity, why is it "new" data schema as opposed to just "data schema 2024"?  It seems that new would very quickly lose its meaning.

Reply:
I just used the word new to reflect it didn't previously exist, but I didn't use that for other files for some reason so not very consistent, so agreed I will update (along with new_data_sample.csv below)